tests/utilities/test_imports.py

```diff
@@ -45,3 +54,48 @@ def test_compare_version(monkeypatch):
     assert not _compare_version("torch", operator.ge, "1.10.0.rc0")
     assert _compare_version("torch", operator.ge, "1.10.0", use_base_version=True)
     assert not _compare_version("torch", operator.ge, "1.10.0")
+
+
+def test_imports():
+    try:
+        import apex  # noqa
+
+        assert _APEX_AVAILABLE
+    except ModuleNotFoundError:
+        assert not _APEX_AVAILABLE
```

Comment:
```suggestion
    try:
        import apex  # noqa
    except ModuleNotFoundError:
        assert not _APEX_AVAILABLE
    else:
        assert _APEX_AVAILABLE

```
lest move it to `else` https://note.nkmk.me/en/python-try-except-else-finally/